splunk-otel-android/src/main/java/io/opentelemetry/rum/internal/OpenTelemetryRumBuilder.java

```diff
@@ -51,7 +52,9 @@ public final class OpenTelemetryRumBuilder {
     private final List<Consumer<InstrumentedApplication>> instrumentationInstallers =
             new ArrayList<>();
 
-    OpenTelemetryRumBuilder() {}
+    OpenTelemetryRumBuilder(SessionId sessionId) {
```

Comment:
Nit: why not just create `SessionId` and `SessionIdTimeoutHandler` here, instead of the `builder()` method?